docs/introduction.rst

```diff
@@ -1,11 +1,43 @@
 Introduction
 ============
 
-Tools for visualizing Rubin Observatory scheduler behaviour and LSST survey status
+``schedview`` is a collection of tools for visualizing Rubin Observatory scheduler behaviour and LSST survey status.
+`RTN-016: Background and concepts for monitoring survey progress and scheduler performance <https://rtn-016.lsst.io/>`_ describes the motivation for the construction of these tools, including some of the use cases for which they are intended and example of current and planned visualizations.
 
-There are presently two different applications in this project:
+The ``schedview`` module includes web applications that supply dashboards designed for the common use cases found in `RTN-016 <https://rtn-016.lsst.io/>`_.
+These web applications can either be run on a surver, where they can be used by multiple users; or locally, including customization with additional plots.
+In addition to the pre-built web applications, the supporting functions and classes are intended to be usable either as parts of other applications, or in jupyter notebooks.
 
-- `sched_maps`, a tool for examing some elements of the state of objects used
-  by the scheduler, particularly those that take the form of a sky map,
-  although some indications of other elements are also present.
-- `prenight`, a pre-night briefing dashboard.
+``schedview`` presently supplies two dashboards:
+
+* ``scheduler_dashboard``, a dashboard for examining the state of a the scheduler an a specific time, intended for using during the night to give insight into the schedulers current behavior.
+  The ``scheduler_dashboard`` shows maps and relevant statistics for all basis functions and rewards for all surveys, given a time and an instance of the scheduler.
+* ``prenight``, a dashboard for preparing for a night of observing.
+  Visualizations included in this dashboard are intended to help schedulers and observing staff understand what the scheduler will to in a night, given the output of a simualtion of the night (as created by `rubin_sim`).
```

Comment:
".. what the scheduler will *do* in a night, given the output of a *simulation* of .. "